src/NzbDrone.Common/Http/HttpClient.cs

```diff
@@ -196,6 +207,7 @@ private HttpResponse ExecuteRequest(HttpRequest request, HttpWebRequest webReque
 
         private HttpResponse ExecuteCurlRequest(HttpRequest request, HttpWebRequest webRequest)
         {
+            _logger.Trace("Curl request");
```

Comment:
I don't see a lot of value with this logging message, something more verbose would be preferred.